contracts/pools/PoolCollection.sol

```diff
@@ -282,8 +283,8 @@ contract PoolCollection is IPoolCollection, Owned, BlockNumber, Utils {
         _poolTokenFactory = initPoolTokenFactory;
         _poolMigrator = initPoolMigrator;
 
-        _setDefaultTradingFeePPM(DEFAULT_TRADING_FEE_PPM);
-        _setNetworkFeePPM(DEFAULT_NETWORK_FEE_PPM);
+        _defaultTradingFeePPM = (DEFAULT_TRADING_FEE_PPM);
```

Comment:
We have usually been emitting events during construction as well (since this is how the backend was built). Are you sure it's no longer required to be this way?

Reply:
(Or is it mostly to reduce the size of the contract?)

Reply:
It is ONLY for that purpose.
I've figured - since this is not an upgradable contract, we may as well "don't care" that no events are emitted in this case (though I do understand that it might actually be a counter-argument for what I am trying to reason).

Reply:
That's a "risky" approach because this will create a problem when moving to a new chain, where the "initial" event has never been seen by the backend. I'll let @yudilevi to revert on this.

Reply:
OK, keep in mind that the context here is V3, which is not very likely to be migrated to another chain.

Reply:
@yudilevi WDYT?

Reply:
I'd try to avoid breaking the convention. Do keep in mind this contract is re-deployed every upgrade (including this one) so these settings will actually get executed and the events will be missing from the live backend

Reply:
The purpose is reducing the size of the contract

Reply:
it will likely break the existing backend - any workaround?

Reply:
Fixed